src/tools/dart-compile.md

```diff
@@ -136,11 +136,21 @@ plus a small [Dart runtime][] that handles
 type checking and garbage collection.
 
 You can distribute and run the output file like you would
-any other executable file:
+any other executable file.
+
+Compile your app and set the output file.
 
 ```terminal
 $ dart compile exe bin/myapp.dart -o /tmp/myapp
+```
+
+When successful, this command returns the following.
+```terminal
 Generated: /tmp/myapp
+```
+
+To run your compiled app, change to the `/tmp` directory and run.
```

Comment:
I think this would be simpler if it was just `/tmp/myapp` without having to change directory.

Reply:
Not a problem here.

Reply:
Agree with mit.